src/query/models/params.go

```diff
@@ -40,6 +48,7 @@ type RequestParams struct {
 	Query      string
 	Debug      bool
 	IncludeEnd bool
+	FormatType FormatType
```

Comment:
Won't this default to 0, which is equivalent to FormatM3QL; this would probably cause issues when trying to parse Prom. Might be easiest to add the prom option as the 0 default option